examples/with-ant-design/next.config.js

```diff
@@ -0,0 +1,8 @@
+const withCss = require('@zeit/next-css')
+
+// fix: prevents error when .css files are required by node
+if (typeof require !== 'undefined') {
+  require.extensions['.css'] = (file) => {}
```

Comment:
This seems incorrect 🤔next-css should also handle the discard of styles on the server.

Reply:
Agreed. But it breaks without this during SSR.

Reply:
Then there must be something wrong with the setup 🤔

Reply:
Or it's related to antd not importing with `.css`

Reply:
Here is what I get:
```
[...]/node_modules/antd/lib/style/index.css:6
@font-face {
^

SyntaxError: Invalid or unexpected token
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:152:10)
    at Module._compile (module.js:624:28)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/luke/binance/react-exchange/node_modules/antd/lib/radio/style/css.js:3:1)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
```

Reply:
@timneutkens they use their custom babel plugin to do it, so no, it's not a direct css import.